database/migration/src/main/resources/changesets/standardize_column_names_patches.xml

```diff
@@ -0,0 +1,64 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+                   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.3.xsd">
+
+    <changeSet author="kshakir" id="standardize_column_names_patches">
+        <comment>
+            Columns now all match standardized naming.
+            FK_CALL_CACHING_HASH_ENTRY_CALL_CACHING_ENTRY_ID must be dropped and re-added so that
+            UC_CALL_CACHING_HASH_ENTRY_CCEI may be renamed.
+        </comment>
```

Comment:
Last time I migrated, metadata was the only table that took hours, 8 to be specific. On the same database, the rest of the schema changes took 8 minutes total. I'm not touching the metadata table, so I don't expect admins to need to wait more than 30 minutes max.